### DISCUSSION TITLE
tabs/ filters in bookmarks

### DISCUSSION THREAD

**Discussion by jackson-digidigital** [dailydotdev/daily#1651] [Feature Requests]
i save a lot of articles for later in bookmark, sometime just content i want to get back to when i need . i have over 30 articles now and its getting kind a hard to find what im looking for . 
it would be nice to have like custom directive inside so i can choose where to save each article.
or maybe even just filtering so i can filter this by the category / tags

**Comment by idoshamun** [comment]
Hi @jackson-digidigital,

Please take a look here https://github.com/dailydotdev/daily/discussions/520

**Reply by jackson-digidigital** [reply]
didnt saw that thanks
